CONTRIBUTING.md

```diff
@@ -120,15 +120,15 @@ We are migrating the resource from SDKv2 to Plugin Framework provider and hence
 
 ### Adding a new resource
 1. Check if the directory for this particular resource exists under `internal/providers/pluginfw/resources`, if not create the directory eg: `cluster`, `volume` etc... Please note: Resources and Data sources are organized under the same package for that service.
-2. Create a file with resource_resource-name.go and write the CRUD methods, schema for that resource. For reference, please take a look at existing resources eg: `resource_quality_monitor.go`
+2. Create a file with resource_resource-name.go and write the CRUD methods, schema for that resource. For reference, please take a look at existing resources eg: `resource_quality_monitor.go`. Make sure to set the user agent in all the CRUD methods. In the `Metadata()`, if the resource is to be used as default, use the method `GetDatabricksProductionName()` else use `GetDatabricksStagingName()` which suffixes the name with `_pluginframework`. 
```

Comment:
This might not be necessary if we decide to incorporate it into a core resource, which I'm currently working on implementing. Just a reminder to myself to update this part of the guidelines if we decide to go that route.